packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -1,12 +1,59 @@
 // @flow
-import React from "react";
+/**
+ * A two-column modal layout.
+ */
+import * as React from "react";
 import {StyleSheet, css} from "aphrodite";
 
-export default class TwoColumnModal extends React.Component<{}> {
+import Color from "wonder-blocks-color";
+
+type Props = {
+    leftContent: React.Node,
+    rightContent: React.Node,
+};
+
+export default class TwoColumnModal extends React.Component<Props> {
     render() {
-        return <div>
-            Hello, world!
-            TODO(mdr): Build TwoColumnModal.
-        </div>;
+        return (
+            <div className={css(styles.container)}>
+                <div className={css(styles.column, styles.leftColumn)}>
+                    {this.props.leftContent}
+                </div>
+                <div className={css(styles.column, styles.rightColumn)}>
+                    {this.props.rightContent}
+                </div>
+            </div>
+        );
     }
 }
+
+const styles = StyleSheet.create({
+    container: {
+        display: "flex",
+        flexDirection: "row",
+        alignItems: "stretch",
+
+        // This minimum height is a fixed number from the spec.
+        minHeight: 464,
```

Comment:
We should also set the `width`, `height` to be a certain percentage or vw/vh amounts.

Reply:
Mm gotcha, looks like the Zeplin spec isn't up-to-date with the proportional numbers you listed in the implementation spec.

Looking back, I suppose we probably shouldn't have dimensions in these layout components at all? Maybe instead we should have `ModalLauncher` set container dimensions, and use Flexbox to stretch the content to fit the container. I imagine that, if we're embedding these layouts directly into a page, we won't want the viewport-dependent sizing.